rebar.config

```diff
@@ -27,30 +27,13 @@
 
 {deps, [
   {sumo_db, "0.6.4"},
-  {riakc, "2.4.1"},
+  {riakc, {git, "https://github.com/basho/riak-erlang-client.git", {tag, "2.5.2"}}},
```

Comment:
@lucafavatella Can we ask maintainers ( @drewkerrigan 😉  ) to publish an updated hex package?
Or use the brand new [@esl](https://hex.pm/users/erlangsolutions) entity in hex to publish esl_riakc and use that package?
Otherwise, we won't be able to publish sumo_db_riak on hex 😢